src/status_im/ui/screens/chat/message/message.cljs

```diff
@@ -612,23 +612,23 @@
    {:keys [on-long-press modal]
     :as   reaction-picker}]
   (let [show-timestamp? (reagent/atom false)]
-    (fn [] [message-content-wrapper message
-            [react/touchable-highlight
-             (when-not modal
-               {:on-long-press
-                (fn [] (on-long-press (if (and outgoing config/delete-message-enabled?)
-                                        [{:on-press #(re-frame/dispatch [:chat.ui/soft-delete-message message])
-                                          :label    (i18n/label :t/delete)
-                                          :id       :delete}]
-                                        [])))
-                :on-press (fn []
-                            (reset! show-timestamp? true))})
-             [react/view (style/message-view-wrapper (:outgoing message))
-              [message-timestamp message show-timestamp?]
-              [react/view {:style (style/message-view message) :accessibility-label :audio-message}
-               [react/view {:style (style/message-view-content)}
-                [message.audio/message-content message] [message-status message]]]]]
-            reaction-picker])))
+    [message-content-wrapper message
```

Comment:
but in that case `show-timestamp?` will be `false` each time , i guess you should move params in `fn[]` instead

Reply:
Hi @flexsurfer, Thank you very much for reviewing PR. I done changes as you requested. Weirdly they changes created exact same result.
1. `show-timestamp?` sets false on re-render, but we are already doing after 2 seconds, for hiding timestamp
2. Even if animation is not complete(time is let's say 20 sec), on rerender animations finished callback is called and sets it to false.

Still, yours is right way to do 👍, Thank you